apps/condo/domains/banking/schema/BankAccount.test.js

```diff
@@ -98,6 +128,39 @@ describe('BankAccount', () => {
                 expect(createdObj.currencyCode).toEqual(readObj.currencyCode)
             })
 
+            test('service can', async () => {
+                const [organization] = await createTestOrganization(adminClient)
+                await createTestBankIntegrationOrganizationContext(adminClient, SBBOLBankIntegration, organization)
+                const [integrationContext] = await createTestBankIntegrationAccountContext(serviceClient, SBBOLBankIntegration, organization)
+
+                const [createdObj] = await createTestBankAccount(serviceClient, organization, {
+                    integrationContext: { connect: { id: integrationContext.id } },
+                })
+                const [readObj] = await BankAccount.getAll(serviceClient, { id: createdObj.id })
+
+                expect(createdObj.organization.id).toEqual(readObj.organization.id)
+                expect(createdObj.tin).toEqual(readObj.tin)
+                expect(createdObj.country).toEqual(readObj.country)
+                expect(createdObj.routingNumber).toEqual(readObj.routingNumber)
+                expect(createdObj.number).toEqual(readObj.number)
+                expect(createdObj.currencyCode).toEqual(readObj.currencyCode)
+            })
+
+            test('service can if BankAccount has no connected integration', async () => {
```

Comment:
Should it really can?
Do we have a case in business-logic of the feature, when service-user is **creating** BankAccount without connected Integration. I know about a case, when service-user is **updating** existing BankAccount without integration…